GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -204,17 +292,23 @@ + (BOOL)supportsSecureCoding {
 - (nullable instancetype)initWithCoder:(NSCoder *)decoder {
   self = [super init];
   if (self) {
-    GIDProfileData *profileData =
-        [decoder decodeObjectOfClass:[GIDProfileData class] forKey:kProfileDataKey];
-    OIDAuthState *authState;
-    if ([decoder containsValueForKey:kAuthState]) { // Current encoding
-      authState = [decoder decodeObjectOfClass:[OIDAuthState class] forKey:kAuthState];
-    } else { // Old encoding
-      GIDAuthentication *authentication = [decoder decodeObjectOfClass:[GIDAuthentication class]
-                                                                forKey:kAuthenticationKey];
-      authState = authentication.authState;
-    }
-    [self updateAuthState:authState profileData:profileData];
+    _refreshTokensHandlerQueue = [[NSMutableArray alloc] init];
```

Comment:
Can we [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up by calling `initWithAuthState:profileData:`?